api/envoy/extensions/load_balancing_policies/client_side_weighted_round_robin/v3/client_side_weighted_round_robin.proto

```diff
@@ -56,4 +56,8 @@ message ClientSideWeightedRoundRobin {
 
   // How often endpoint weights are recalculated. Default is 1 second.
   google.protobuf.Duration weight_update_period = 5;
+
+  // The penalty used to adjust utilization with errors per second (eps).
```

Comment:
How is an error defined here?

Reply:
It will come from the backend.  See the `eps` field being added to the ORCA proto in https://github.com/cncf/xds/pull/58.

Reply:
> Thanks, a few questions:
> 
> 1. Will this be implemented in Envoy?
> 2. Is there somewhere we can capture the intuition of why this is needed and useful? The PR message explains where it's needed, but not the underlying reason of why you would make this kind of change.

Also shared with you our design review slides.

Reply:
Can we just define here what an error is for HTTP and gRPC? I think it's just helpful to be precise and clear at the point of use.

Reply:
The design leaves it to the service to collect and report qps/eps along with CPU utilization. I slightly updated the comment to clarify that collection is up to the endpoint.

Reply:
That doesn't answer the question of what an error is for the purpose of computing EPS.

Reply:
Updated, PTAL.